### PR TITLE
Option pass flaky list json oo

### PR DESCRIPTION
**Pull request by winglam**
Please review https://github.com/TestingResearchIllinois/iFixFlakies/pull/8 first.

### CODE REVIEW

**Comment by winglam** [comment]
@august782 Please review. kthxbai.

File: src/main/java/edu/illinois/cs/dt/tools/minimizer/MinimizerPlugin.java [hunk 1]
```diff
@@ -48,22 +51,34 @@ public MinimizerPlugin(final InstrumentingSmartRunner runner) {
...
     }
 
-    private Stream<TestMinimizer> fromDtList(final Path path, MavenProject project) {
-        TestPluginPlugin.info("Creating minimizers for file: " + path);
+    private Stream<TestMinimizer> fromDtList(Path path, MavenProject project) {
+        if (FLAKY_LIST != null) {
+            path = Paths.get(FLAKY_LIST);
+            TestPluginPlugin.info("dt.minimizer.flaky.list argument specified: " + FLAKY_LIST);
+        }
+        TestPluginPlugin.info("Creating minimizers for file: " + path);
```
Comment (august782) [inline src/main/java/edu/illinois/cs/dt/tools/minimizer/MinimizerPlugin.java]: There's some huge whitespace here at the end of this line that should be removed.

Comment (winglam) [reply src/main/java/edu/illinois/cs/dt/tools/minimizer/MinimizerPlugin.java]: Done.

**Review by august782 — APPROVED** [review]
LGTM

**Pull request merged**